migrations/versions/0359_remove_sched_notifications_.py

```diff
@@ -0,0 +1,35 @@
+"""
+
+Revision ID: 0359_remove_sched_notifications
```

Comment:
I'm afraid I've just merged a migration number 0359, so this will need to be updated as a result